### ISSUE TITLE
Make usage sentences to go from easiest to hardest

### ISSUE THREAD

**Issue by RomanTymchyshyn** [Uran198/words-telegram-bot#12]
IDEA:
	* For each word calculate the number of times it is used.
	* When the word is queried, choose an example with the most common words.
	* Can separately preprocess the data for hungarian.

**Comment by RomanTymchyshyn** [comment]
@Uran198, do you mean ordering the sentences within a specific "card"? If so would that be really useful?
Also, it seems like this one (#11) is really useful and if the system allows you to define/change cards, it should be easy to reorder sentences as well, manually though.

**Comment by attila0x2A** [comment]
> do you mean ordering the sentences within a specific "card"?

Yes, for a single word there are often much more than 3 usage sentences, this is about choosing which ones of them to show.

> If so would that be really useful?

Sometimes examples are very complicated, this might solve it.
